curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/661f48f412d7631a1d9c30e6.md

```diff
@@ -0,0 +1,159 @@
+---
+id: 661f48f412d7631a1d9c30e6
+title: Step 75
+challengeType: 0
+dashedName: step-75
+---
+
+# --description--
+
+You should use `console.log()` to print the result of calling the `highPrecedence` function with the string `"5*3"`.
+
+# --hints--
+
+You should call `console.log()`.
+
+```js
+
+assert.match(code, /console\.log\(.*\);/);
+
+```
+
+You should call your `highPrecedence` function.
+
+```js
+
+assert.match(code, /console\.log\(highPrecedence\((?:"[^"]*"|[^)]*)\)\);/);
+
+
+```
+
+Pass `5*3` as the argument
+
+```js
+
+assert.match(code, /console\.log\(highPrecedence\("5\*3"\)\);/);
+
```

Comment:
And this works fine:
```suggestion
assert.match(code, /console\.log\(highPrecedence\(["']5\*3["']\)\);?/);
```

We should allow single quotes, and make the semi-colon optional.